### DISCUSSION TITLE
Get event not triggerred when the request is interrupted

### DISCUSSION THREAD

**Discussion by nodarai** [minio/minio#18806] [Q&A] [answered]
Hello,

I have configured MinIO server to send the `get` (s3:ObjectAccessed:*) events to a webhook and it works great except the case when the connection is interrupted.

For example, if I have a bucket `test` and a large file `5g.blob` and I start the download with 
```bash
wget http://127.0.0.1:9000/test/5g.blob
```
If I do Ctrl+c and interrupt the download before the finish the `get` event is not notified on the webhook.
Is that by the design? If yes, can you provide the reasoning why is that the case?

P.S: In the trace there is a record that file was accessed

**Comment by harshavardhana** [comment, marked as answer]
Events only generate after a success, that is the nature of bucket events.
